src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1119,35 +1119,54 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_AddSrpService(c
 
     srpService->mService.mPort = aPort;
 
-    // Check if there are some optional text entries to add.
-    if (aTxtEntries && aTxtEntriesSize != 0)
+#if OPENTHREAD_API_VERSION >= 132
```

Comment:
@Damian-Nordic Happy to accomodate this new feature, but can you advise to which OpenThread commit you bumped for this ? At your side this get's updated through Zephyr I assume.
btw I see another check for API_VERSION >= 126 in OpenThreadUtils.cpp - is the feature expected to work with anything in between ?

Reply:
We bumped OpenThread in Zephyr to 48b129 (API_VERSION == 134). The check for API_VERSION >= 126 was added because with that API version OpenThread switched from otMasterXXX to otNetworkXXX function naming.

Reply:
@Damian-Nordic Thanks - we will align to that commit to avoid OpenThread inconsistencies